### ISSUE TITLE
Provide newer version of xamarin-android.oss (as artifact)

### ISSUE THREAD

**Issue by tenzap** [xamarin/xamarin-android#7739]
### Android application type

Classic Xamarin.Android (MonoAndroid12.0, etc.)

### Affected platform version

Mono

### Description

Here you link to a place to download xamarin-oss which points to build artifacts.
https://github.com/xamarin/xamarin-android#current
It points to 
https://dev.azure.com/xamarin/public/_build/results?buildId=54953&view=results

Last build artifacts are from Feb/2022.

These are too old to build a project targetting v13.0 or API33

Could you provide a newer version for xamarin.android-oss ?

That's the error we get:
```
         /home/runner/xamarin.android-oss/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/Xamarin.Android.Legacy.targets(265,5): error XA0000: Could not determine API level for $(TargetFrameworkVersion) of 'v13.0'. [/home/runner/work/keepass2android/keepass2android/src/keepass2android/keepass2android-app.csproj]
```

### Steps to Reproduce

-

**Comment by marcus-nordstrom-p** [comment]
@jpobst is this solved yet? Can't seem to build with v13.0

**Comment by pjcollins** [comment]
We are unfortunately no longer producing Linux packages for classic Xamarin.Android. We do however provide a Linux SDK as part of the `android` .NET workload. I would recommend installing .NET 7 or .NET 8 RC and running the `dotnet workload install android` [command](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-workload-install) to work with our latest SDK, as the Xamarin.Android SDK is near end of life. The .NET 7 SDK supports API 33 (v13.0), and the .NET 8 SDK supports API 34.

Instructions for upgrading your project from Xamarin.Android to .NET can be found here https://learn.microsoft.com/en-us/dotnet/maui/migration/native-projects.
